infra/Pos/Reporting/Methods.hs

```diff
@@ -118,23 +117,20 @@ ipv4Local w =
 
 -- | Retrieves node info that we would like to know when analyzing
 -- malicious behavior of node.
-getNodeInfo :: (MonadDHT m, MonadIO m) => m Text
+getNodeInfo :: (MonadIO m) => m Text
```

Comment:
Oops, I should have left a note for myself. I removed this early on intending to restore it. Will fix it up.